### PR TITLE
Feature#46

### PR DESCRIPTION
**Pull request by perlajarillo**
A progressBar indicates to the user that something is happening when a new search is in progress.

### CODE REVIEW

File: weather-app/src/main/java/rcgonzalezf/org/weather/location/LocationManager.java [hunk 1]
```diff
@@ -87,13 +89,21 @@ public void disconnect() {
     }
   }
 
-  @VisibleForTesting void onEmptyLocation(BaseActivity baseActivity, View content) {
+  @VisibleForTesting void onEmptyLocation(final BaseActivity baseActivity, View content) {
     if (baseActivity != null && content != null) {
       Snackbar.make(content, baseActivity.getString(R.string.location_off_msg),
           Snackbar.LENGTH_SHORT).show();
+      //After 5 seconds the activity will be recharged
+        new Handler().postDelayed(new Runnable(){
```
Comment (moxi) [inline weather-app/src/main/java/rcgonzalezf/org/weather/location/LocationManager.java]: Why are you adding a 5 seconds delay?

File: weather-app/src/main/res/layout/weather.xml [hunk 2]
```diff
@@ -10,12 +10,15 @@
...
     >
 
+
   <android.support.design.widget.CoordinatorLayout
       android:id="@+id/content"
       android:layout_width="match_parent"
       android:layout_height="match_parent"
       android:orientation="vertical"
       >
+
```
Comment (moxi) [inline weather-app/src/main/res/layout/weather.xml]: Try to run the autoformat tool, this is going to help the reviewers to focus on code changes and not new empty lines, spaces or style changes.

**Comment by moxi** [comment]
I'll take a look on the branch and possibly I'll make some changes :), thanks for the help.

File: weather-app/src/main/java/rcgonzalezf/org/weather/location/LocationManager.java (hunk 1 above)
Comment (perlajarillo) [inline weather-app/src/main/java/rcgonzalezf/org/weather/location/LocationManager.java]: I am adding 5 seconds delay to refresh the activity looking for an available Internet connection. I am wondering if  there is another way to make a better user experience.

File: weather-app/src/main/res/layout/weather.xml (hunk 2 above)
Comment (perlajarillo) [inline weather-app/src/main/res/layout/weather.xml]: Ok, I will keep that in mind.

**Comment by moxi** [comment]
Thanks for the help, I'm merging this to develop.

**Pull request merged**